### ISSUE TITLE
Can’t play videos on yotter.xyz

### ISSUE THREAD

**Issue by SwiftWinds** [pluja/Parasitter#203]
**Describe the bug**
Upon clicking on a video, I am greeted with this screen:

![image](https://user-images.githubusercontent.com/12981958/113430875-c5055700-938f-11eb-8523-1841705b64e5.png)

**To Reproduce**
1. Go to yotter.xyz 
2. Search for any video (e.g., "chubbyemu")
3. Click on the first video result

**Expected behavior**
For the UI to watch the video to appear.

**Screenshots**
See above.

**Smartphone (please complete the following information):**
 - OS: iOS 14.3
 - Browser: Safari
 - Version: 14.3 ([the iOS version is the same as the Safari version](https://browserhow.com/how-to-check-current-installed-version-of-apple-safari/#check-current-version-of-safari-browser-on-ios))

**Additional context**
Thank you so much for your free and open-source, privacy-focused web app! I use it all the time. :)

**Comment by obeho** [comment]
Same error here. Since many days. All videos have the same problem.

**Comment by FireMasterK** [comment]
It's an instance issue, works on my instance 🤔

**Comment by obeho** [comment]
Yeah, maybe some rate-limiting issue on the IP.

**Comment by SwiftWinds** [comment]
https://yotter.kavin.rocks works for me. However, I'm wary that it's hosted in India, which is around the globe from where I am in the states, so it might be a bit slow at times for me. 

Do you know if Yotter can be hosted with Google Cloud Run?

**Comment by obeho** [comment]
> https://yotter.kavin.rocks works for me

I think its hosted on Cloudflare, so you should have no problem about the latencies, wherever you are.

**Comment by FireMasterK** [comment]
> https://yotter.kavin.rocks works for me. However, I'm wary that it's hosted in India, which is around the globe from where I am in the states, so it might be a bit slow at times for me.

That's true, unfortunately, you would have a latency of at least 270ms, multi-region load balancing isn't supported by Yotter you would have to look for a US instance, or host one yourself.

Alternatively, could check my project - [Piped](https://piped.kavin.rocks/), which does support multi-region loadbalancing. (and has a server in the US!)

>Do you know if Yotter can be hosted with Google Cloud Run?

I don't think this is possible as Yotter would require at least 3 processes to work - nginx, python/gunicorn, and the ytproxy.

> I think its hosted on Cloudflare, so you should have no problem about the latencies, wherever you are.

This isn't exactly true always, uncached requests still have to travel to India, here, every page is uncached, only images, CSS, and JS are cached.

**Comment by swgrrt** [comment]
I have the same issue on my private instance which is only used by me. So I guess the problem is not caused by IP rate limiting.

**Comment by FireMasterK** [comment]
Yes, this is likely due to YouTube's new consent cookie, thinking about it now.

**Comment by lewis149** [comment]
But wouldn't that affect Invidious too?

**Comment by FireMasterK** [comment]
> But wouldn't that affect Invidious too?

They have a fix for that: https://github.com/iv-org/invidious/commit/739f6105073245b051aa08354e9a4f15c25a3572

**Comment by lewis149** [comment]
Now it redirects to some hosting website

**Comment by pluja** [comment]
I am aware of this, I will be solving these problems on the following months when I finish my final exams!

**Comment by lewis149** [comment]
Thanks and good luck
